tests/foreman/installer/test_installer.py

```diff
@@ -324,7 +325,8 @@ def sat_fapolicyd_install(module_sat_ready_rhels):
     assert install_satellite(sat, installer_args, enable_fapolicyd=True).status == 0, (
         "Satellite installation failed (non-zero return code)"
     )
-    if settings.server.is_ipv6:
+    # TODO(jpathan): Check whether this is valid for dualstack
```

Comment:
In dual stack it can directly access the CDN so no HTTP proxy is needed. Technically the best description is `if not has_ipv4: enable_proxy()`.

Reply:
Keeping it as is, and I will remove the TODO comment(s) then. 

`if not has_ipv4: enable_proxy()` is indeed an ideal description, but `settings.server.network_type == NetworkType.IPV6: enable_proxy()` is as close as we can get with our current implementation.

Reply:
I thought about expanding `NetworkType` with methods:
```python
@property
def has_ipv4(self):
    return self in (NetworkType.IPV4, NetworkType.DUALSTACK)

def has_ipv6(self):
    return self in (NetworkType.IPV6, NetworkType.DUALSTACK)
```

In the end I didn't end up suggesting it but it may be a better way of thinking about it.

Reply:
Left as https://github.com/SatelliteQE/robottelo/pull/17904#discussion_r2068365084 since this inline comment was outdated.